src/Files.App/Views/Properties/HashesPage.xaml.cs

```diff
@@ -21,7 +21,10 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
 		{
 			var np = (MainPropertiesPage.PropertyNavParam)e.Parameter;
 			if (np.navParameter is ListedItem listedItem)
-				HashesViewModel = new(listedItem);
+			{
+				HashesViewModel = new();
+				HashesViewModel.Initialize(listedItem);
```

Comment:
Please do this in constructor of the ViewModel

Reply:
Done.